input/data/2013/2013ApJ...764...38A/tev-000001.yaml

```diff
@@ -1,4 +1,25 @@
 source_id: 1
 reference_id: 2013ApJ...764...38A
 
-# TODO: add data
+data:
+  livetime: 41.25
+  significance: 7.5
+
+pos:
+  ra: {val: 0h6m26s, err: 0.09d, err_sys: 50s}
+  dec: {val: 72d59m01.0s, err: 0.04, err_sys: 50s}
+
+morph:
+  type: gauss
+
+  sigma: {val: 0.30, err: 0.03}
+  sigma2: {val: 0.24, err: 0.03}
+  # TODO: frame {radec, galactic} information is missing in the paper
+  pa: {val: 17.4, err: 15.8}
+
+spec:
+  type: pl
+
+  norm: {val: 0.091, err: 0.031, err_sys: 0.017}
```

Comment:
I think there's a typo in the norm err. Should be 0.013, now is 0.031, no? (see abstract)